barretenberg/cpp/src/barretenberg/boomerang_value_detection/graph_description_sha256.test.cpp

```diff
@@ -14,39 +14,65 @@
 
 using namespace bb;
 using namespace bb::stdlib;
+using namespace cdg;
 
 using Builder = UltraCircuitBuilder;
+using byte_array_pt = byte_array<Builder>;
+using packed_byte_array_pt = packed_byte_array<Builder>;
+using field_pt = field_t<Builder>;
 
-using byte_array_ct = byte_array<Builder>;
-using packed_byte_array_ct = packed_byte_array<Builder>;
-using field_ct = field_t<Builder>;
+/**
+ static analyzer usually prints input and output variables as variables in one gate. In tests these variables
+ are not dangerous and usually we can filter them by adding gate for fixing witness. Then these variables will be
+ in 2 gates, and static analyzer won't print them. functions fix_vector and fix_byte_array do it
+ for vector of variables and packed_byte_array respectively
+*/
+
+void fix_vector(std::vector<field_pt>& vector)
+{
+    for (auto& elem : vector) {
+        elem.fix_witness();
+    }
+}
+
+void fix_byte_array(packed_byte_array_pt& input)
+{
+    std::vector<field_pt> limbs = input.get_limbs();
+    fix_vector(limbs);
+}
 
 /**
- all these tests check graph description for sha256 circuits. All circuits have to consist from 1 connected component
+ all these tests check graph description for sha256 circuits. All circuits have to consist of 1 connected component
  */
 
 TEST(boomerang_stdlib_sha256, test_graph_for_sha256_55_bytes)
```

Comment:
It's better to add separate descriptions to each test. If I go to the last one, I don't know that there is a description here.